generic.go

```diff
@@ -533,3 +533,51 @@ func KeyBy[V any, K comparable](slice []V, f func(v V) K) map[K]V {
 	}
 	return result
 }
+
+// PadRigth will fill the slice with the given value and the specified size.
+// No padding will take place if the absolute value of the given size is less
+// than or equal to the length of the array
+func PadRight[V any](slice []V, size int, pad V) []V {
+	if size <= len(slice) {
```

Comment:
Shouldn't you take the absolute value of size with math.Abs?
Also, len(slice) is technically to the right. Shouldn't this be `size < len(slice) `?

Reply:
Well, size is a... Size, not an index. So nvm the second part

Reply:
In `PadLeft` and `PadRight` `size` is always expected to be positive

Reply:
The sign is only used in `Pad` to indicate which direction you want to pad.

Reply:
But in this case, if you pass a negative value, it won't take any effect. Shouldn't 10 and -10 have the same output? I'm asking mostly due to the description. I'm fine with both ways.

Reply:
yea, I copied this description from `Pad`, it shouldn't have the "absolute" part 😅